src/nft/pages/profile/profile.tsx

```diff
@@ -47,42 +79,38 @@ const ProfileContent = () => {
   const cartExpanded = useBag((state) => state.bagExpanded)
   const isNftListV2 = useNftListV2Flag() === NftListV2Variant.Enabled
 
+  const pageWidthAdjustment =
+    cartExpanded && (!isNftListV2 || sellPageState === ProfilePageStateType.VIEWING)
+      ? SHOPPING_BAG_WIDTH
+      : isNftListV2 && sellPageState !== ProfilePageStateType.VIEWING
```

Comment:
I think this should be stored in a const like `var check = isNftListV2 && sellPageState !== ProfilePageStateType.VIEWING` then it could also be used above on line 83 as `!check`